modules/ROOT/pages/storage.adoc

```diff
@@ -88,6 +88,40 @@ storage:
       with_mount_unit: true
 ----
 
+
+.Defining a disk with multiple partitions
+In this example, we wipe the disk and create two new partitions. 
+
+[source,yaml]
+----
+variant: fcos
+version: 1.2.0
+storage:
+  disks:
+    -
+      # Mandatory. We use the World-Wide Number ID of the drive to ensure
```

Comment:
Hmm, the comments are substantive, but they're inconsistent with the other examples on this page...

Reply:
...as in move to a "Note" above it?

Reply:
So, wipe all embedded comments?